app/components/optimized-power-select/options/template.hbs

```diff
@@ -21,6 +21,7 @@
     @renderAll={{this.isTesting}}
     as |option index|
   >
+    {{!template-lint-disable  require-context-role}}
```

Comment:
Why not set a `role`?

Reply:
```bash
 ember-template-lint --fix:
/home/falk/Dev/timed-frontend/app/components/optimized-power-select/options/template.hbs
  30:6  error  You have an element with the role of "option" but it is missing the required (immediate) parent element of "[listbox]". Reference: https://www.w3.org/TR/wai-aria-1.1/#option.  require-context-role
```
It actually has the role, but ember-template-lint is erroring incorrect, as it doesn't know about the `vertical-collection` wrapper.